geotrellis/geotrellis-site
# Transition to a Jekyll based development environment

## Overview

Remove everything that wasn't in `/static` (Scala services, Nginx, old STRTA, Travis config, etc), and replace with the supporting Vagrant and Docker resources necessary to provide a Ruby 2.4.6 Jekyll-based development environment.

Resolves #87 

## Testing Instructions

Following the instructions in the `README`, ensure that you can make tweaks to the site and see your changes reflected locally by hitting http://localhost:4000.
